src/common/utilities.ts

```diff
@@ -176,3 +176,21 @@ export async function updateKedroVizPanel(lsClient: LanguageClient | undefined):
     const projectData = await executeGetProjectDataCommand(lsClient);
     KedroVizPanel.currentPanel?.updateData(projectData);
 }
+
+export async function isKedroProject(): Promise<boolean> {
```

Comment:
Is there a reason why you use `async` method but using `readFileSync` and `existsSync`